enhancements/installer/openstack-customer-provided-subnets.md

```diff
@@ -0,0 +1,131 @@
+Another example of this split is found in companies that provide public access to their OpenStack resources, giving their tenants/customers quota on compute and storage resources but providing pre-defined networks for them.
+In these scenarios, only OpenStack administrators will manage these networks, which will be generally provider networks, (as opposed to tenant networks), since they require physical network infrastructure configuration (typically VLANs). So when OpenStack tenants log in they will find them pre-created, shared with them and ready to use.
+To accomplish this, our infrastructure needs an OpenStack account access that can be separated into "networking infrastructure" versus "application". Networking infrastructure includes elements related to the networks (networks, subnets, routing tables, internet gateways, NAT, VPN). Application items include OpenStack resources directly touching nodes within the cluster (LBs, security groups, Swift containers, nodes). This is a typical separation of responsibilities among IT groups and deployment/provisioning pipelines within large companies.
+
+## Motivation
+
+### Goals
+As an administrator, I would like to create or reuse my own networks, subnets and routing schemes that I can deploy OpenShift to.
+
+### Non-Goals
+There are no changes in expectations regarding publicly addressable parts of the cluster.
+
+## Proposal
+- Installer allow users to provide a list of subnets that should be used for the cluster. Since there is an expectation that networking is being shared, the installer cannot modify the networking setup, but changes required to the shared resources like Tags that do not affect the behavior for other tenants of the network will be made.
+- The installer validates the assumptions about the networking setup
+- Non-networking infrastructure resources that are specific to the cluster and resources owned by the cluster will be created by the installer. So resources like security groups, roles, RHCOS boot images, and ignition storage objects.
+- The infrastructure resources owned by the cluster continue to be clearly identifiable and distinct from other resources.
+- Destroying a cluster must make sure that no resources are deleted that didn't belong to the cluster. Leaking resources is preferred over any possibility of deleting non-cluster resources.
+
+
+## User Stories
+As an administrator of a cluster using provider networking, I have a pre-configured provider network, that cannot be manipulated by openstack tenants, and I want to install openshift on it.
+
+## Implementation Details Notes/Constraints
+
+### Installer Resources
+The user provides a list of subnets to the installer in the openstack platform section of the install-config.yaml.
```

Comment:
If as stated above " provider network, that cannot be manipulated by openstack tenants", 
then perhaps the user will also have to provide a list of ip addresses to use?